log/brimapi/client.go

```diff
@@ -0,0 +1,103 @@
+package brimapi
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/allegro/akubra/service"
+	"github.com/hashicorp/consul/api"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	uploadSynctasksURI         = "/v1/processes/uploadsynctasks"
+	serviceDiscoverySchemeName = "service"
+	httpRequestTimeout         = time.Second * 3
+	httpConsulRequestTimeout   = time.Second * 2
+)
+
+var (
+	discoveryServices *service.Services
+	httpClient        *http.Client
+)
+
+func init() {
```

Comment:
what is being initialized here?

Reply:
`discoveryServices` instance with related `httpClient` + `consulConfig`